src/main/kotlin/app/revanced/patches/youtube/misc/playeroverlay/PlayerOverlaysHookPatch.kt

```diff
@@ -1,23 +1,30 @@
-package app.revanced.patches.youtube.misc.playeroverlay.patch
+package app.revanced.patches.youtube.misc.playeroverlay
 
-import app.revanced.patcher.annotation.Description
-import app.revanced.patcher.annotation.Name
 import app.revanced.patcher.data.BytecodeContext
 import app.revanced.patcher.extensions.InstructionExtensions.addInstruction
 import app.revanced.patcher.patch.BytecodePatch
-import app.revanced.patcher.patch.annotations.DependsOn
-import app.revanced.patches.youtube.misc.integrations.patch.IntegrationsPatch
-import app.revanced.patches.youtube.misc.playeroverlay.annotation.PlayerOverlaysHookCompatibility
+import app.revanced.patcher.patch.annotation.CompatiblePackage
+import app.revanced.patcher.patch.annotation.Patch
+import app.revanced.patches.youtube.misc.integrations.IntegrationsPatch
 import app.revanced.patches.youtube.misc.playeroverlay.fingerprint.PlayerOverlaysOnFinishInflateFingerprint
 
-@Name("Player overlays hook")
-@Description("Hook for adding custom overlays to the video player.")
-@PlayerOverlaysHookCompatibility
-@DependsOn([IntegrationsPatch::class])
-class PlayerOverlaysHookPatch : BytecodePatch(
-    listOf(
-        PlayerOverlaysOnFinishInflateFingerprint
-    )
+@Patch(
+    description = "Hook for adding custom overlays to the video player.",
+    dependencies = [IntegrationsPatch::class],
+    compatiblePackages = [
+        CompatiblePackage("com.google.android.youtube", [
+            "18.16.37",
+            "18.19.35",
+            "18.20.39",
+            "18.23.35",
+            "18.29.38",
+            "18.32.39"
+        ])
+    ]
+)
```

Comment:
❓ : Missing patch name

```suggestion
@Patch(
    name = "Player overlays hook",
    description = "Hook for adding custom overlays to the video player.",
    dependencies = [IntegrationsPatch::class],
    compatiblePackages = [
        CompatiblePackage("com.google.android.youtube", [
            "18.16.37",
            "18.19.35",
            "18.20.39",
            "18.23.35",
            "18.29.38",
            "18.32.39"
        ])
    ]
)
```